src/app/space/forge-wizard/components/single-input/single-input.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="form-group" [class.has-danger]="!form.valid" [ngSwitch]="field.valueType" [formGroup]="form">
+  <label title="{{field.label}}" [class.field-required]="field.required" [for]="field.name"
+         class="user-select-none col-xs-12 col-sm-4">{{field.label}}</label>
+  <div class="col-sm-8">
+    <input *ngSwitchCase="'java.lang.String'" id="{{field.name}}" title="{{field.label}}" name="{{field.name}}"
```

Comment:
You really should have `label`s for each of these `input` fields, but you can do it in another issue.